### ISSUE TITLE
[Support]: Cannot run valet commands

### ISSUE THREAD

**Issue by s-aqua** [github/gh-valet#89]
### What operating system are you using?

- [ ] Linux
- [ ] macOS
- [X] Windows

### What version of the tool are you using?

gh valet  v0.1.17

### What happened?

Hey team

I tried running gh valet audit, forecast, migrate etc and the command just hangs and doesn't do anything. And it does not provide any errors or logs.

Tried on 2 separate computers that are having the same issues and I also tried WSL Ubuntu on those machines and the commands work.

I hope its something that I may have overlooked, not sure if there is something I am missing, but I have everything required installed on both computers.

Any help would be appreciated.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dwhathaway** [comment]
@s-aqua to start, can you confirm you've completed all the [configuration](https://github.com/github/gh-valet#configuration) steps documented here?

Also, if you could run the following steps:

1. Run `docker image ls` and confirm that you see `ghcr.io/valet-customers/valet-cli   latest` in the list of images?
2. Run `gh version` and provide the output
3. Run `gh valet version` and provide the output

**Comment by s-aqua** [comment]
Hey @dwhathaway yep, I have configured valet and can run the update

>gh valet update
Login Succeeded
latest: Pulling from valet-customers/valet-cli
Digest: sha256:d0891245ce92cd7e4864c1a799a5cd67d8eea03f235e2d337549fed81a3695ca
Status: Image is up to date for ghcr.io/valet-customers/valet-cli:latest
ghcr.io/valet-customers/valet-cli:latest


1. 
![image](https://user-images.githubusercontent.com/57583354/198153466-6cdc36f0-e447-4252-a7eb-40a3f77280a8.png)


2. > gh version
gh version 2.17.0 (2022-10-04)
https://github.com/cli/cli/releases/tag/v2.17.0

3. > gh valet version
gh version 2.17.0 (2022-10-04)
https://github.com/cli/cli/releases/tag/v2.17.0
gh valet        github/gh-valet v0.1.17
valet-cli       v0.1.0.14014

**Comment by dwhathaway** [comment]
@s-aqua I'm investigating if changes in https://github.com/github/gh-valet/pull/86 caused some issues with stream output on Windows.  While I'm investigating that, can you downgrade to the `v0.1.16` release of `gh-valet` and see if it has the same behavior?

To install an older version, you'll first need to remove the extension, then install the specific release using the `--pin` flag.

```
gh extension remove github/gh-valet
gh extension install github/gh-valet --pin v0.1.16
```

> Note: using the `--pin` flag will cause `gh-valet` to be permanently pinned so future updates will require again uninstalling/re-installing the `gh-valet` extension to get to the latest.

**Comment by s-aqua** [comment]
@dwhathaway I have downgraded valet to v0.1.16 and it works.

Thanks.

**Comment by dwhathaway** [comment]
Great, thank you for confirming @s-aqua.  I'm glad to hear v0.1.6 works as expected.  We'll look into this the output streaming logic.  In the meantime, please remain on v0.1.6 if it is working for you you.

**Comment by dwhathaway** [comment]
@s-aqua is fixed in the [v0.1.18](https://github.com/github/gh-valet/releases/tag/v0.1.18) release.  Thank you for identifying it.

Note that you'll need to fully uninstall the current version of `gh-valet` using `gh extension remove gh-valet`, and then run  `gh extension install github/gh-valet` to install the latest release.
